.github/workflows/verify_library_generation.yaml

```diff
@@ -10,21 +10,35 @@ on:
 name: verify_library_generation_against_googleapis-gen
 jobs:
   verify_library_generation:
-    runs-on: ubuntu-22.04
     strategy:
       matrix:
         java: [ 8 ]
+        os: [ ubuntu-22.04, macos-12 ]
+    runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-java@v3
       with:
         java-version: ${{ matrix.java }}
         distribution: temurin
         cache: maven
+    - uses: kanga333/variable-mapper@v0.3.0
```

Comment:
From go/github-actions we may want to specify a commit hash of this third party tool instead of a version tag

Reply:
done